frontend/src/pages/answer/Answer.jsx

```diff
@@ -0,0 +1,124 @@
+import React, { useContext } from 'react';
```

Comment:
This message is interesting. It has a query graph and a full knowledge graph, but the results are empty. Makes sense that the results table doesn't show up. The kg bubble graph doesn't show up because it relies on the results to get node bubble sizes and therefore isn't showing up either (This seems expected behavior to me). I can add a message in the UI saying that there are no results for this message so it's clear the UI didn't "break".

Reply:
Added no results message in 28e84e975b3a613682d903cba2e1558f156e689f

Reply:
The fix looks good, but I want to consider options for displaying the KG without results. See #194.